doc/Start.md

```diff
@@ -1,46 +1,194 @@
-![](img/start-http-server.png)
-- You own a full client
-#### Game Server
-- You need a game server that is compatible with the original game. There are many implementations/versions/forks that are compatible, you can use any of them, but we suggest using one of the following two, because we test using these emulators:
-  - [rAthena](https://github.com/rathena/rathena)
-  - [Hercules](https://github.com/HerculesWS/Hercules/)
-- All client/packet versions are supported, but the number of missing features increases with higher dates, since we need time to get everything implemented. We advise you to use versions older than `2015` for the best experience, but this is not a restriction, only a suggestion.
+
+### 2.1 Prerequisites
+
+Here's a list of things that you will need to have for a sucessfull installation and gameplay of your favorite server.
+
+| Item            | Description                                                                   |
+|-----------------|-------------------------------------------------------------------------------|
+| RoBrowserLegacy | Client to Run Ragnarok Online on browser                                      |
+| NodeJS 20+      | NodeJS to compile RoBrowser assets                                            |
+| wsproxy         | A Websocket Proxy to interligate Login + Server + Map into one endpoint       |
+| Game Server     | Your Emulator that usually is rAthena or Hercules                             |
+| Game Files      | A kRO or any RO installation files to use as your base for the server         |
+| Browser         | Any browser that is compatible with OpenGL ES 2.0 (Requirement for RoBrowser) |
+| Embed Server    | Any programming language that has embed servers to use. Example: PHP or Ruby  |
+
+
+To run roBrowser you will need an up to date browser that supports [WebGL](http://www.chromeexperiments.com/webgl/) and is OpenGL ES 2.0 compatible. We've tested the following browsers:
+* Chrome: _(Desktop & Mobile)_
+* FireFox: _(Desktop & Mobile)_
+* Safari: _(Desktop & Mobile)_
+* Edge: _(Desktop)_
+
+Others will probably work as well especially if **Chromium based**, but there might be slight differences.
+
+### 3. Setup Instructions
+
+
+#### 3.1 Setup: NodeJS and NVM
+
+> If you already have NodeJS LTS installed, skip this step.
+
+Make sure you're running NodeJS LTS or 18 above. Validate your version using the command:
+
+```shell
+node --version
+# v20.10.0 (example output)
+```
+
+If your version is minor than 18 you can 
+
+To install NVM you can use the command below:
+
+```shell
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+```
+
+After you install NVM, reopen your terminal and check if the installation was successful:
+
+```bash
+nvm --version 
+# 0.39.0
+```
+
+Now you need to switch to the appropriate version to run MeteorJS 2 by using the command:
```

Comment:
I assume you just copied this, remove the mention to meteorjs

Reply:
My bad. I used this same lines in a JS tutorial related to Meteor and just copy/pasted.